www/about/policies/terms-of-service.md

```diff
@@ -38,7 +38,7 @@ these Terms.
 1. General Terms
 
   1. The Service is a platform to facilitate payments for goods or services
-     ("Work").
+     ("Work"). The scope of Work is defined by Projects.
```

Comment:
Nice. Fixed in https://github.com/gratipay/gratipay.com/pull/4117/commits/b332633238497f6a97f45f72ad4194d9fe11a1a1.